yearzero/YZRoll.js

```diff
@@ -119,6 +119,30 @@ class YZRoll {
 			.length;
 	}
 
+	/**
+	 * The quantity of ones(banes) on base dice.
+	 * @type {number}
+	 * @readonly
+	 */
+	get baseBaneCount() {
```

Comment:
I think a cleaner method for this getter could be:
```js
return this.count('<base|ammo>', 1);
```

Reply:
I think so, really I just copied from a previous example and modified them. Happy to make and test those changes!

Reply:
One other thing that was caught during my game last week is the 6s on the gear dice were not persistent during pushes, unlike the 1s and successes on the main dice. So need to look into that too!